### ISSUE TITLE
Schedules: Invalid search term entered. GET returned: 500 A server error has occurred. 

### ISSUE THREAD

**Issue by j-insan3** [ansible/awx#1100]
##### ISSUE TYPE
 - Bug Report

##### COMPONENT NAME
 - API
 - UI

##### SUMMARY
Just updated my awx environment using docker-compose. Since then it is showing this error when openeing the schema's page.

##### ENVIRONMENT
* AWX version: 1.0.2.92
* AWX install method: docker on linux
* Ansible version:  2.4.2.0 
* Operating System: CentOS 7 server, ubuntu 17.10 Desktop
* Web Browser: Firefox / Chrome

##### STEPS TO REPRODUCE

Log in to AWX and click Schedules

##### EXPECTED RESULTS

Show schedules

##### ACTUAL RESULTS

Invalid search term entered. GET returned: 500 A server error has occurred.

##### ADDITIONAL INFORMATION

awx_task log:

```
192.168.127.28 - - [01/Feb/2018:12:05:20 +0000] "GET /api/v2/unified_jobs/?page_size=20&order_by=-finished&not__launch_type=sync HTTP/1.1" 200 41347 "http://awx.int.previder.nl/" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:58.0) Gecko/20100101 Firefox/58.0" "-"
[pid: 29|app: 0|req: 41/159] 192.168.127.28 () {46 vars in 2273 bytes} [Thu Feb  1 12:05:19 2018] GET /api/v2/unified_jobs/?page_size=20&order_by=-finished&not__launch_type=sync => generated 41347 bytes in 436 msecs (HTTP/1.1 200) 9 headers in 251 bytes (1 switches on core 0)
2018-02-01 12:05:20,371 ERROR    django.request Internal Server Error: /api/v2/schedules/
Traceback (most recent call last):
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/core/handlers/base.py", line 249, in _legacy_get_response
    response = self._get_response(request)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/utils/decorators.py", line 185, in inner
    return func(*args, **kwargs)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/usr/lib/python2.7/site-packages/awx/api/generics.py", line 253, in dispatch
    return super(APIView, self).dispatch(request, *args, **kwargs)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/rest_framework/views.py", line 489, in dispatch
    response = self.handle_exception(exc)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/rest_framework/views.py", line 449, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/rest_framework/views.py", line 486, in dispatch
    response = handler(request, *args, **kwargs)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/rest_framework/generics.py", line 201, in get
    return self.list(request, *args, **kwargs)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/rest_framework/mixins.py", line 45, in list
    return self.get_paginated_response(serializer.data)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/rest_framework/serializers.py", line 738, in data
    ret = super(ListSerializer, self).data
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/rest_framework/serializers.py", line 262, in data
    self._data = self.to_representation(self.instance)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/rest_framework/serializers.py", line 656, in to_representation
    self.child.to_representation(item) for item in iterable
  File "/usr/lib/python2.7/site-packages/awx/api/serializers.py", line 3103, in to_representation
    ret = super(LaunchConfigurationBaseSerializer, self).to_representation(obj)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/rest_framework/serializers.py", line 500, in to_representation
    ret[field.field_name] = field.to_representation(attribute)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/rest_framework/fields.py", line 1816, in to_representation
    return method(value)
  File "/usr/lib/python2.7/site-packages/awx/api/serializers.py", line 299, in _get_related
    return {} if obj is None else self.get_related(obj)
  File "/usr/lib/python2.7/site-packages/awx/api/serializers.py", line 3949, in get_related
    if obj.unified_job_template.project:
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/polymorphic/models.py", line 159, in accessor_function
    attr = model.base_objects.get(pk=self.pk)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/db/models/query.py", line 380, in get
    self.model._meta.object_name
DoesNotExist: Project matching query does not exist.
2018-02-01 12:05:20,371 ERROR    django.request Internal Server Error: /api/v2/schedules/
Traceback (most recent call last):
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/core/handlers/base.py", line 249, in _legacy_get_response
    response = self._get_response(request)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/utils/decorators.py", line 185, in inner
    return func(*args, **kwargs)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/usr/lib/python2.7/site-packages/awx/api/generics.py", line 253, in dispatch
    return super(APIView, self).dispatch(request, *args, **kwargs)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/rest_framework/views.py", line 489, in dispatch
    response = self.handle_exception(exc)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/rest_framework/views.py", line 449, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/rest_framework/views.py", line 486, in dispatch
    response = handler(request, *args, **kwargs)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/rest_framework/generics.py", line 201, in get
    return self.list(request, *args, **kwargs)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/rest_framework/mixins.py", line 45, in list
    return self.get_paginated_response(serializer.data)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/rest_framework/serializers.py", line 738, in data
    ret = super(ListSerializer, self).data
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/rest_framework/serializers.py", line 262, in data
    self._data = self.to_representation(self.instance)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/rest_framework/serializers.py", line 656, in to_representation
    self.child.to_representation(item) for item in iterable
  File "/usr/lib/python2.7/site-packages/awx/api/serializers.py", line 3103, in to_representation
    ret = super(LaunchConfigurationBaseSerializer, self).to_representation(obj)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/rest_framework/serializers.py", line 500, in to_representation
    ret[field.field_name] = field.to_representation(attribute)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/rest_framework/fields.py", line 1816, in to_representation
    return method(value)
  File "/usr/lib/python2.7/site-packages/awx/api/serializers.py", line 299, in _get_related
    return {} if obj is None else self.get_related(obj)
  File "/usr/lib/python2.7/site-packages/awx/api/serializers.py", line 3949, in get_related
    if obj.unified_job_template.project:
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/polymorphic/models.py", line 159, in accessor_function
    attr = model.base_objects.get(pk=self.pk)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/db/models/query.py", line 380, in get
    self.model._meta.object_name
DoesNotExist: Project matching query does not exist.
[pid: 26|app: 0|req: 30/160] 192.168.127.28 () {46 vars in 2343 bytes} [Thu Feb  1 12:05:20 2018] GET /api/v2/schedules/?page_size=20&order_by=unified_job_template__polymorphic_ctype__model&next_run__isnull=false => generated 41 bytes in 57 msecs (HTTP/1.1 500) 6 headers in 203 bytes (1 switches on core 0)
192.168.127.28 - - [01/Feb/2018:12:05:20 +0000] "GET /api/v2/schedules/?page_size=20&order_by=unified_job_template__polymorphic_ctype__model&next_run__isnull=false HTTP/1.1" 500 41 "http://awx.int.previder.nl/" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:58.0) Gecko/20100101 Firefox/58.0" "-"
192.168.127.28 - - [01/Feb/2018:12:05:20 +0000] "OPTIONS /api/v2/unified_jobs/ HTTP/1.1" 200 9802 "http://awx.int.previder.nl/" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:58.0) Gecko/20100101 Firefox/58.0" "-"
[pid: 25|app: 0|req: 59/161] 192.168.127.28 () {50 vars in 2264 bytes} [Thu Feb  1 12:05:20 2018] OPTIONS /api/v2/unified_jobs/ => generated 9802 bytes in 72 msecs (HTTP/1.1 200) 9 headers in 250 bytes (1 switches on core 0)
```

**Comment by mabashian** [comment]
I can confirm that I'm seeing this behavior as well

**Comment by twhiston** [comment]
I am also seeing this with both schedules, the workflow editor and portal mode

**Comment by ryanpetrello** [comment]
This should be resolved via 81bdbef785 on `devel`.
